automation_tools/satellite6/upgrade/client.py

```diff
@@ -174,12 +174,12 @@ def satellite6_client_setup():
                 host=sat_host
             )
         # Refresh subscriptions on clients
-        time.sleep(5)
+        time.sleep(30)
```

Comment:
I think its ok, its just a one time sleep just before 'yum clean all' on all the clients. However this is not something 'number of clients' * 30. Its just one single 30 seconds.